python/cuml/tests/test_compose.py

```diff
@@ -94,6 +94,9 @@ def test_column_transformer(
     "transformer_weights", [None, {"scaler": 2.4, "normalizer": 1.8}]
 )
 @pytest.mark.parametrize("sparse_threshold", [0.2, 0.8])
+@pytest.mark.filterwarnings(
+    "ignore::numba.core.errors.NumbaPerformanceWarning"
+)
```

Comment:
This one is in the `pyproject.toml` filters list (there's a few other places too).